x/tier/types/keys.go

```diff
@@ -89,8 +89,8 @@ func LockupKeyToAddresses(key []byte) (sdk.AccAddress, sdk.ValAddress) {
 	return delAddr, valAddr
 }
 
-// LockupKeyToAddressesAtHeight retreives delAddr, valAddr, and creationHeight from provided unlocking Lockup key.
-func LockupKeyToAddressesAtHeight(key []byte) (sdk.AccAddress, sdk.ValAddress, int64) {
```

Comment:
suggestion: Are the `LockupKeyToAddressesAtHeight` not still handy util functions to have around?